docs/video-processing/functionalities/set-project-options.md

```diff
@@ -11,9 +11,11 @@ nav_order: 1
 
```

Comment:
We should mention that the multi-color/multi channel mode (for >= 3 ) is basically a beta version. It has never been tested for real-world scenarios.